resources/definitions/fdmprinter.def.json

```diff
@@ -1787,6 +1787,18 @@
                     "limit_to_extruder": "infill_extruder_nr",
                     "settable_per_mesh": true
                 },
+                "infill_scaling_z":
+                {
+                    "label": "Infill Vertical Scaling",
+                    "description": "Scales the vertical infill pattern. Values larger than 100% stretch the pattern, values less than 100% compress the pattern.",
```

Comment:
"Scales the infill pattern **vertically**"
(There is no such thing as the 'vertical infill pattern')

Please explain why a user would use this. E.g. "Values larger than 100% make the infill more stiff in the vertical direction."

Reply:
Somebody asked me to implement it because they thought it would be useful. I did so but can't offer any justification for it other than it can make interesting variations in the gyroid infill.

![Screenshot_2019-09-14_09-25-52](https://user-images.githubusercontent.com/585618/67483140-0f43be00-f65d-11e9-8f3d-a779d4bae841.png)